webrender/src/device/gfx/buffer.rs

```diff
@@ -11,6 +11,7 @@ use std::mem;
 
 pub const MAX_INSTANCE_COUNT: usize = 8192;
 pub const TEXTURE_CACHE_SIZE: usize = 128 << 20; // 128MB
+const MINIMUM_ALIGNMENT_MASK: usize = 3;
```

Comment:
This could be `BUFFER_COPY_ALIGNMENT_MASK` and a link for the spec would be helpful as well.

Reply:
ok, great, let's do that!